scss/navbar.scss

```diff
@@ -0,0 +1,26 @@
+.navbar {
+    overflow: hidden;
+    background-color: white;
+    border-bottom: 1px solid black;
+    position: fixed; 
+    top: 0; 
+    height: $nav-bar-height;
+    width: 100%;
+    z-index: 9999;
```

Comment:
`z-index: 1;` is enough I think (unless you have something else in z-index 1) keeping low values makes it easier to override and to find a problem if it occurs.